packages/firestore/src/local/simple_db.ts

```diff
@@ -173,15 +172,14 @@ export class SimpleDb {
       .catch(error => {
         // Abort the transaction if there was an error.
         transaction.abort(error);
+        return PersistencePromise.reject<T>(error);
```

Comment:
I'm having trouble figuring out what this return value affects, if anything.  My guess is that it doesn't affect anything, because we'll have aborted the transaction and so transaction.completionPromise will be a failed promise below and we won't use transactionFnResult.  If that's correct, maybe add a comment to this effect just so it's clear how the error is expected to be propagated.

Reply:
Done.